packages/sdk-js/src/GrowthBook.ts

```diff
@@ -1298,6 +1298,18 @@ export class GrowthBook<
         undo.push(mutate.declarative(mutation as DeclarativeMutation).revert);
       });
     }
+    if (changes.urlRedirect) {
+      if (this._ctx.navigate) {
+        this._ctx.navigate(changes.urlRedirect);
+      } else {
+        const script = document.createElement("script");
+        script.innerHTML = `setTimeout(() => { window.location.replace("${
```

Comment:
Do we need to do a white overlay over the page or something similar so the user isn't exposed to the control during the timeout period?